### DISCUSSION TITLE
I have student developer pack but I get this message:  Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.

### DISCUSSION THREAD

**Discussion by MATTYGILO** [community/community#46045] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I have this message:  Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.

But I have student developer pack, but copilot has stopped working

**Comment by SilasRodrigues19** [comment]
Hey @MATTYGILO 

I found 2 answered discussions on this topic, maybe it can help you:

https://github.com/community/community/discussions/43695
https://github.com/community/community/discussions/44054

**Comment by MATTYGILO** [comment, marked as answer]
So the solution for me was

> I needed to update all my Intelij copilot plugins
